ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml

```diff
@@ -3,28 +3,16 @@ base_images:
     name: "4.7"
     namespace: ocp
     tag: base
-  ocp_builder_rhel-7-golang-1.13:
-    name: builder
-    namespace: ocp
-    tag: rhel-7-golang-1.13
-  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.6
   ocp_builder_rhel-8-golang-1.15-openshift-4.7:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.15-openshift-4.7
-  ocp_builder_rhel-8-golang-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
 binary_build_commands: make build --warn-undefined-variables
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
```

Comment:
I recommend you use the same build tag as your builder image -- rhel-8-golang-1.15-openshift-4.7

Reply:
updated in current push, thanks!